apps/payment/models.py

```diff
@@ -33,26 +33,46 @@ class Payment(models.Model):
     )
 
     content_type = models.ForeignKey(ContentType)
+    """Which model the payment is created for. For attendance events this should be event."""
     object_id = models.PositiveIntegerField()
+    """
+    Object id for the model chosen in content_type.
+    For events this should be the event id and not attendance event id.
```

Comment:
The attendance event id will always be equal to the event id, since the primary key of an attendance event is an event.
https://github.com/dotkom/onlineweb4/blob/develop/apps/events/models.py#L399

Reply:
Ah, that makes sense.